build.gradle

```diff
@@ -228,6 +260,8 @@ dependencies {
   implementation 'org.postgresql:postgresql:42.6.0'
   runtimeOnly 'org.hsqldb:hsqldb'
 
```

Comment:
If we added H2 below I think we should remove hsqldb

Reply:
done. slightly out of scope of this ticket though, so i might get a complaint!